pkg/container/qemu_runner.go

```diff
@@ -717,7 +717,25 @@ func createMicroVM(ctx context.Context, cfg *Config) error {
 	// panic=-1 ensures that if the init fails, we immediately exit the machine
 	// Add default SSH keys to the VM
 	sshkey := base64.StdEncoding.EncodeToString(pubKey)
-	baseargs = append(baseargs, "-append", kernelConsole+" nomodeset random.trust_cpu=on panic=-1 sshkey="+sshkey+" melange_qemu_runner=1")
+
+	// Build kernel command line arguments
+	kernelArgs := kernelConsole + " nomodeset random.trust_cpu=on panic=-1 sshkey=" + sshkey + " melange_qemu_runner=1"
+
+	// Check for TESTING environment variable and pass it to microvm-init
+	// TESTING must be a number (or empty string)
+	if testingValue, ok := os.LookupEnv("TESTING"); ok {
+		if testingValue == "" {
+			log.Debugf("qemu: TESTING env set to empty string, passing to microvm-init via kernel cmdline")
+			kernelArgs += " melange.testing="
+		} else if _, err := strconv.Atoi(testingValue); err == nil {
+			log.Debugf("qemu: TESTING env set to %s, passing to microvm-init via kernel cmdline", testingValue)
```

Comment:
I'm not **adamant** about this but probably we want this at an informational level, rather than debug, given that it's an input into the build environment, and it makes it easier to capture than running `--log-level=debug`. 

(OTOH, if we start to capture build logs then the dmesg/syslog/journal would contain the actual passed command line flags, so it might be redundant. But we don't do that yet.)